models/stitch/transform/zendesk_org_url.sql

```diff
@@ -7,5 +7,10 @@ with tickets as (
 )
 
 select distinct
-  split_part(split_part(api_url, '//', 2), '.zendesk.com', 1) as org_url_identifier
+  {{ 
+    dbt_utils.split_part(
+      dbt_utils.split_part('api_url', "'//'", 2
+      ), "'zendesk.com'", 1
```

Comment:
I think you missed a `.` when translating this -- the original was `'.zendesk.com'` :)